opentelemetry-api/src/opentelemetry/_logs/_internal/__init__.py

```diff
@@ -99,8 +65,26 @@ def __init__(
         self._attributes = attributes
 
     @abstractmethod
-    def emit(self, record: "LogRecord") -> None:
-        """Emits a :class:`LogRecord` representing a log to the processing pipeline."""
+    def is_enabled(
+        self,
+        severity_number: Optional[SeverityNumber] = None,
+        context: Optional[Context] = None,
+    ) -> bool:
+        """Returns True if the logger is enabled for given severity and context, False otherwise."""
+
+    @abstractmethod
+    def emit(
+        self,
+        name: str = None,
```

Comment:
Is the spec missing the name parameter
https://opentelemetry.io/docs/specs/otel/logs/api/#emit-a-logrecord ? BTW in the sdk you are using `event_name`

Reply:
it's just got merged - https://github.com/open-telemetry/opentelemetry-specification/pull/4320 will be released soon

> BTW in the sdk you are using event_name

Good catch!